### ISSUE TITLE
Yubikey 5C NFC not detected anymore on Windows

### ISSUE THREAD

**Issue by magratheaner** [keepassxreboot/keepassxc#10603]
## Overview

Noticed yesterday that my YubiKey 5C NFC is not detected anymore by KeepassXC on Win11. Whenever I try to refresh the hardware keys in the UI, I get "No hardware keys found".
The setup was working before. The key is still working on my Android phone with Keepass2Android and on my Macbook with KeepassXC 2.7.7. Restarted the PC multiple times, used different USB ports, upgraded to and reinstalled KeepassXC ver. 2.7.7., YubiKey Manager is detecting the key. 

## Steps to Reproduce

1. Slot in YubiKey 5C NFC anywhere on Win11 with KeePassXC 2.7.7
2. YubiKey Manager detects the key
3. Open KeepassXC, refresh hardware key list
4. Get message "No hardware keys found"

## Expected Behavior

Hardware key found, can be used to unlock database with challenge response

## Actual Behavior

Message "No hardware keys found", can't unlock database

## Context

Since I haven't found anyone else with this issue, I think this must correlate with a recent Windows update or some system configuration on my part. Still should never happen.

### Debug info:

KeePassXC - Version 2.7.7
Revision: 68e2dd8

Qt 5.15.11
Debugging mode is disabled.

Operating system: Windows 11 Version 2009
CPU architecture: x86_64
Kernel: winnt 10.0.22631

Enabled extensions:
- Auto-Type
- Browser Integration
- Passkeys
- SSH Agent
- KeeShare
- YubiKey
- Quick Unlock

Cryptographic libraries:
- Botan 3.1.1

**Comment by droidmonkey** [comment]
My usual go-to is that your challenge response port was reset, but it works on other platforms so that is not it. You could try going to the device manager and seeing if windows is not fully detecting the various hardware key components for whatever reason. 

You can also try the previous release as a portable zip to see if it is an actual bug: https://github.com/keepassxreboot/keepassxc/releases/download/2.7.6/KeePassXC-2.7.6-Win64.zip

**Comment by magratheaner** [comment]
It just resolved itself. It's now working with ver. 2.7.7 again. I looked into the device manager and whenever I plugged the key in, it sometimes showed a smartcard and sometimes didn't. Also I used "update driver" on the smartcard in device manager, but it said that I was already using the correct driver. So I didn't really change anything.

In the end it might have been a Windows update or some sort of system instability on my end. 

Sorry for bothering, can be closed.
